kumascript/macros/HTMLRefTable.ejs

```diff
@@ -17,41 +17,47 @@ let l10n = {
         "en-US": "Element",
         "fr"   : "Élément",
         "zh-CN": "元素",
-        "ru"   : "Элемент"
+        "ru"   : "Элемент",
+        "ko"   : "요소"
```

Comment:
In this case, you can move `ru` before `zh-CN` and `ko` after `fr`, but only because this is a single line.